request.js

```diff
@@ -778,6 +778,8 @@ Request.prototype.start = function () {
           var e = new Error('ESOCKETTIMEDOUT')
           e.code = 'ESOCKETTIMEDOUT'
           e.connect = false
+          e.host = self.host
+          e.port = self.port
```

Comment:
That is bizarre, it looks like it's a number everywhere else. See https://github.com/seegno-forks/request/blob/88f9a04572c2fec21bf46ab2628950c3fae8bf20/request.js#L303-L314